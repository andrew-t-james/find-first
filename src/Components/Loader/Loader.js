import React, { Component } from 'react';
import { ScaleLoader } from 'halogenium';

class Loader extends Component {
  render() {
    return (
      <ScaleLoader color="#1e90ff" height="4rem"  />
    );
  }
}

export default Loader;