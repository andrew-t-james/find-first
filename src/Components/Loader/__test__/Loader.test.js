import React from 'react';
import { shallow } from 'enzyme';
import Loader from '../Loader';

describe('<Loader />', () => {
  let wrapper;

  beforeEach(() => wrapper = shallow(<Loader />));

  test('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});
