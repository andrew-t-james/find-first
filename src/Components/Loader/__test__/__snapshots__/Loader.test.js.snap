// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> should match snapshot 1`] = `
<ScaleLoader
  color="#1e90ff"
  height="4rem"
  loading={true}
  margin="2px"
  radius="2px"
  width="4px"
/>
`;
